shared/pollTypes.py

```diff
@@ -67,11 +67,11 @@ def toDict(self):
         """
         out = {}
 
-        out["question"] = self.question
+        out["question"] = self.question.toDict()
         out["responses"] = []
 
         for response in self.responses:
-            out["responses"].append(response)
+            out["responses"].append(response.toDict())
```

Comment:
question is a string and does not implement toDict().  This crashes at runtime.
The question should just be a string because it is being placed into a dict.

out {
       "question": question <-- question refers to the string variable
       "responses": []
}

This is what out will look like.  So it does not make sense to convert the string variable to a dict because it is being placed into a dict

Reply:
I see what you are saying, but I believe question is _not_ a string. It should be a PollQuestion object.

Reply:
This is how you created the poll.  You asked for string input and use it to create the Poll object.

def prompt_for_poll():
    question = input("Enter your poll question: ")
    poll = Poll(question)
    return poll

Reply:
Have you pulled the latest code?

Reply:
https://github.com/harrismcc/classroom-voter/blob/2a9b10993102bd46515696211c0f3757b12501a9/professor.py#L10

Reply:
Ok that was the problem.